README.md

```diff
@@ -2,15 +2,64 @@
 
 A plugin for [Nautobot](https://github.com/nautobot/nautobot) [Chatops Plugin](https://github.com/nautobot/nautobot-plugin-chatops/)
 
-## Installation
-
-### Build Status
+## Build Status
 
 | Branch      | Status |
 |-------------|------------|
 | **main** | [![Build Status](https://www.travis-ci.com/nautobot/nautobot-plugin-chatops-ansible.svg?token=D7kytCzfCypoGoueSBqJ&branch=main)](https://www.travis-ci.com/github/nautobot/nautobot-plugin-chatops-ansible) |
 | **develop** | [![Build Status](https://www.travis-ci.com/nautobot/nautobot-plugin-chatops-ansible.svg?token=D7kytCzfCypoGoueSBqJ&branch=develop)](https://www.travis-ci.com/github/nautobot/nautobot-plugin-chatops-ansible) |
 
+## Summary
+
+Chatbot that allows users to communicate directly with Ansible AWX/Tower from various chat platforms.
+
+The Nautobot Chatops Ansible plugin extends the capabilities of the Nautobot Chatops plugin to include a new command available to the plugin. This is done by registering to the Python entry point in Nautobot Plugin Chatops, that provides functionality to the code written to interact with Ansible. This plugin introduces the following sub commands to the `ansible` command:
+
+* get_dashboard
+* get_inventory
+* get_projects
+* get_job_templates
+* run_job_template
+* get_jobs
+
+### Get Dashboard
+
+The dashboard command provides an Ansible Tower/AWX status dashboard. This gives a summary of:
+* How many hosts are on the Tower system
+* How many failed hosts (last job is failed)
+* Total number of inventories
+* How many inventories have failed
+* How many project sync failures have occurred
+
+### Get Inventory
+
+The inventory sub-command provides for having inventory details. Providing what hosts are in what inventory.
```

Comment:
This sentence did not read well to me.

The inventory sub-command provides a list of hosts in the selected inventory.